src/traces/table/plot.js

```diff
@@ -0,0 +1,892 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var c = require('./constants');
+var d3 = require('d3');
+var gup = require('../../lib/gup');
+var Drawing = require('../../components/drawing');
+var extendFlat = require('../../lib/extend').extendFlat;
+var svgUtil = require('../../lib/svg_text_utils');
+var raiseToTop = require('../../lib').raiseToTop;
+var cancelEeaseColumn = require('../../lib').cancelTransition;
+
+module.exports = function plot(gd, calcdata) {
+
+    if(c.clipView) {
+        gd._fullLayout._paper.attr('height', 2000);
+    }
+
+    var table = gd._fullLayout._paper.selectAll('.table')
+        .data(calcdata.map(gup.unwrap), gup.keyFun);
+
+    table.exit().remove();
+
+    table.enter()
+        .append('g')
+        .classed('table', true)
+        .attr('overflow', 'visible')
+        .style('box-sizing', 'content-box')
+        .style('position', 'absolute')
+        .style('left', 0)
+        .style('overflow', 'visible')
+        .style('shape-rendering', 'crispEdges')
+        .style('pointer-events', 'all'); // todo restore 'none'
```

Comment:
Why would we need to restore `'none'` here?

Reply:
I'm on the fence about `pointer-events` - right now, one can't copy-paste cell text, as its results would be interesting in cases of LaTeX or other formatted cell content. But maybe a better choice would be to enable copying text in which case `'none'` won't be good.

`'none'` is good if table cell contents are not meant to be copied, ie. the user only scrolls up/down and rearranges columns. During development, I rarely activate `'none'` because it makes Dev Tools element selection nigh impossible; `todo` is an easy to find tagging of what to revisit.

Reply:
Ok, thanks for the info.

Reply:
Folks will definitely want to copy text. I wouldn't worry about LaTeX, as long as it's MathJax it's just paths anyway.